src/modules/job-exec/job-exec.c

```diff
@@ -158,6 +158,32 @@ static struct jobinfo * jobinfo_new (void)
     return job;
 }
 
+flux_future_t *jobinfo_shell_rpc_pack (struct jobinfo *job,
+                                       const char *topic,
```

Comment:
commit message "Add jobinfo_shell_rpc_pack() as a wrapper around flux_rpc_pack() which sends to a shell topic on the first rank of a job."  Not wrong, but reads funny to me, maybe a little run on?

"flux_rpc_pack().  Function will send an RPC to the designated shell topic on the first rank of a job."?